src/Casdoor.Client/Extensions/CasdoorOptionsExtension.cs

```diff
@@ -227,14 +233,19 @@ public static CasdoorOptions Validate(this CasdoorOptions options)
                 new StaticConfigurationManager<OpenIdConnectConfiguration>(options.Protocols
                     .OpenIdConnectConfiguration);
         }
-
-        string metadataAddress = options.Protocols.Authority;
-        if (metadataAddress.EndsWith("/", StringComparison.Ordinal) is false)
+        else
         {
-            metadataAddress += "/";
+            string metadataAddress = options.Protocols.Authority;
+            if (metadataAddress.EndsWith("/", StringComparison.Ordinal) is false)
```

Comment:
you could do it shorter :)
```csharp
metadataAddress.TrimEnd('/') + "/.well-known/openid-configuration"
```
and 243 line not needed

Reply:
Done, Thanks~